test/__mocks__/tempCustomModule.ts

```diff
@@ -3,7 +3,7 @@ class CustomApiProvider {
     return 'custom-api-provider';
   }
 
-  async callApi(prompt: string) {
+  async callApi(_prompt: string) {
```

Comment:
can you revert these (and below) or move them to a new PR with a linter rule change?